WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ActivityIndicatorView.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+/// This is a wrapper view for using the UIActivityIndicator in SwiftUI. The native progress indicator is not available under iOS 13.
+///
+struct ActivityIndicatorView: UIViewRepresentable {
```

Comment:
I think `SwiftUI` tends to drop the `View` postfix for the types.
`ActivityIndicator` seems a bit more aligned with their naming conventions. What do you think?

Reply:
I agree, I changed the name e7a0b72cab56ff5918c92327a66403a6403b3156